src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs

```diff
@@ -9,12 +9,22 @@ public class RabbitMQClusterTransport : RabbitMQTransport
         /// Creates new instance of the RabbitMQ transport to connect to a RabbitMQ cluster.
         /// </summary>
         /// <param name="topology">The custom topology to use.</param>
-        /// <param name="connectionString">Connection string.</param>
+        /// <param name="connectionString">Connection string of the nodes.</param>
         /// <param name="queueMode">The queue mode for receiving queues.</param>
         /// <param name="delayedDeliverySupportConfiguration">The timeouts configuration.</param>
         public RabbitMQClusterTransport(Topology topology, string connectionString, QueueMode queueMode, DelayedDeliverySupport delayedDeliverySupportConfiguration)
             : base(GetBuiltInTopology(topology), connectionString, queueMode, delayedDeliverySupportConfiguration == DelayedDeliverySupport.UnsafeEnabled)
         {
         }
+
+        /// <summary>
+        /// Adds a new node for use within a cluster.
+        /// </summary>
+        /// <param name="host">The hostname of the node.</param>
+        /// <param name="port">The port of the node.</param>
+        public void AddClusterNode(string host, int port = 5672)
```

Comment:
I'm ok with this, but I'm still a bit unhappy about having to use a connection string for the first node and then this API for all further nodes. Feels weird somehow.

Reply:
Given the API we have to work with, I can only assume there's some reason why the rabbit client only allows for these options to change. Fun note: this would never work with AMQP, so if your broker is an AMQP broker, you _HAVE_ to use a load balancer.